packages/turbo-codemod/__tests__/set-default-outputs.test.ts

```diff
@@ -61,8 +61,8 @@ describe("add-default-outputs", () => {
     expect(output.pipeline["build-one"].outputs).toStrictEqual(["foo"]);
     expect(output.pipeline["build-two"].outputs).toStrictEqual(undefined);
     expect(output.pipeline["build-three"].outputs).toStrictEqual([
-      "dist/**/*",
-      "build/**/*",
+      "dist/**",
+      "build/**",
```

Comment:
These are actually equivalent in doublestar land, but we use `**` everywhere, and I don't want to introduce confusion by all of a sudden swapping to this new glob